core/modules/query/ValueExpr.cc

```diff
@@ -74,7 +74,7 @@ renderList(QueryTemplate& qt, ValueExprPtrVector const& vel) {
 ////////////////////////////////////////////////////////////////////////
 
 std::ostream& operator<<(std::ostream& os, const ValueExpr::FactorOp& factorOp) {
-    os << "FactorOp(op:";
+    os << "FactorOp(";
     switch(factorOp.op) {
     case ValueExpr::NONE: os << "NONE"; break;
```

Comment:
`case` should be indented as per:
https://developer.lsst.io/cpp/style.html#a-switch-statement-should-have-the-following-form